lazer/sdk/rust/client/src/client.rs

```diff
@@ -1,7 +1,52 @@
+//! # Pyth Lazer Client
+//!
+//! This module provides a high-level client for connecting to Pyth Lazer data streams.
+//! The client maintains multiple WebSocket connections for redundancy and provides
+//! automatic deduplication of messages.
+//!
+//! ## Features
+//!
+//! - Multiple redundant WebSocket connections
+//! - Automatic message deduplication
+//! - Exponential backoff for reconnections
+//! - Configurable timeouts and channel capacities
+//! - Builder pattern for easy configuration
+//!
+//! ## Basic Usage
+//!
+//! ```rust,ignore
```

Comment:
I think it's better to have an example that at least compiles and use `no_run` instead of `ignore`.